src/main/java/org/generationcp/middleware/service/api/SampleListService.java

```diff
@@ -66,6 +66,8 @@ SampleList moveSampleList(final Integer sampleListId, final Integer newParentFol
 
 	List<SampleListDTO> getSampleLists(final List<Integer> datasetIds);
 
+	List<SampleListDTO> getSampleListsByStudy(Integer studyId, boolean withGenotypesOnly);
```

Comment:
sometimes it gets hard to read boolean params. In this case is not only filtering by study but by another boolean param that tends to make our code dirty. A function searchSampleLists could work better, but we can do it later, no need to change it now